src/core/print_ps.cpp

```diff
@@ -258,7 +258,8 @@ CalcAllValues(bool PrintLandscape, bool PrintDoCont, bool overview, double PageW
 		width *= fieldwidth/fullwidth;
 		height *= fieldheight/fullheight;
 	}
-	return { width, height, real_width, real_height, field_x, field_y, field_w, field_h, stage_field_x, stage_field_y, stage_field_w, stage_field_h, step_size, spr_step_size, step_width };
+	return std::tuple<float, float, float, float, float, float, float, float, float, float, float, float, float, float, short>(
```

Comment:
So far I haven't been able to find a good way around this since it looks like Visual Studio uses a tuple class with an 'explicit' constructor.
http://msdn.microsoft.com/en-us/library/bb982837.aspx
I'm pretty sure that's why the braced initializer lists haven't been working.
If you have any good ideas here, let me know.